app/hooks/scenarios/index.ts

```diff
@@ -1140,3 +1140,41 @@ export function useDeletePUScenaro() {
     },
   });
 }
+
+export function useDownloadSolutionsSummary({
+  requestConfig = {
```

Comment:
I would keep it simple, I don't think you are going to change the `requestConfig` property at any point and we avoid sending empty objects from it is invoked.